moveit_ros/manipulation/move_group_pick_place_capability/src/pick_place_action_capability.cpp

```diff
@@ -576,7 +472,6 @@ void move_group::MoveGroupPickPlaceAction::fillGrasps(moveit_msgs::PickupGoal &g
                                                  -std::numeric_limits<double>::max());
     }
     goal.possible_grasps.push_back(g);
```

Comment:
You were right, I just changed that to trigger the check...
For some reason my clang-format (3.9) keeps reordering my list of includes alphabetically,
so I don't use it at the moment...